portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -445,6 +462,14 @@ definition {
 			expected = "${expectedValue}");
 	}
 
+	macro assertResponseHasCorrectInformation {
```

Comment:
in this assert we are asserting speciifically name of an object entry so we may want to name the macro in a more detailed way, it will make the test more readable on the .testcase layer